backend/bang-ggood/src/main/java/com/bang_ggood/global/exception/ExceptionCode.java

```diff
@@ -1,103 +1,101 @@
-    QUESTION_DIFFERENT(HttpStatus.BAD_REQUEST, "수정할 질문 목록이 기존의 질문 목록과 동일하지 않습니다."),
+    QUESTION_ID_ERROR(HttpStatus.INTERNAL_SERVER_ERROR, BangggoodCode.QUESTION_ERROR, "중복된 질문 ID가 존재해 질문을 생성할 수 없습니다."),
+    QUESTION_HIGHLIGHT_ERROR(HttpStatus.INTERNAL_SERVER_ERROR, BangggoodCode.QUESTION_ERROR, "잘못된 하이라이트 키워드가 존재해 질문을 생성할 수 없습니다."),
+    QUESTION_INVALID(HttpStatus.BAD_REQUEST, BangggoodCode.QUESTION_ERROR, "잘못된 질문 ID입니다."),
+    QUESTION_DUPLICATED(HttpStatus.BAD_REQUEST, BangggoodCode.QUESTION_ERROR, "중복된 질문이 존재합니다."),
+    QUESTION_DIFFERENT(HttpStatus.BAD_REQUEST, BangggoodCode.QUESTION_ERROR, "수정할 질문 목록이 기존의 질문 목록과 동일하지 않습니다."),
 
     // User
-    USER_NOT_FOUND(HttpStatus.UNAUTHORIZED, "유저가 존재하지 않습니다."),
-    USER_EMAIL_ALREADY_USED(HttpStatus.CONFLICT, "이미 해당 이메일을 사용하는 유저가 존재합니다."),
-    GUEST_USER_NOT_FOUND(HttpStatus.INTERNAL_SERVER_ERROR, "게스트 유저가 존재하지 않습니다."),
-    GUEST_USER_UNEXPECTED_EXIST(HttpStatus.INTERNAL_SERVER_ERROR, "예상치 못한 게스트 유저가 존재합니다. 데이터베이스를 확인해주세요."),
+    USER_NOT_FOUND(HttpStatus.UNAUTHORIZED, BangggoodCode.USER_NOT_FOUND, "유저가 존재하지 않습니다."),
+    USER_EMAIL_ALREADY_USED(HttpStatus.CONFLICT, BangggoodCode.USER_EMAIL_ALREADY_USED, "이미 해당 이메일을 사용하는 유저가 존재합니다."),
+    GUEST_USER_NOT_FOUND(HttpStatus.INTERNAL_SERVER_ERROR, BangggoodCode.GUEST_USER_NOT_FOUND, "게스트 유저가 존재하지 않습니다."),
+    GUEST_USER_UNEXPECTED_EXIST(HttpStatus.INTERNAL_SERVER_ERROR, BangggoodCode.GUEST_USER_UNEXPECTED_EXIST, "예상치 못한 게스트 유저가 존재합니다. 데이터베이스를 확인해주세요."),
 
     //Email
-    EMAIL_INVALID_FORMAT(HttpStatus.BAD_REQUEST, "유효하지 않은 이메일 형식입니다."),
+    EMAIL_INVALID_FORMAT(HttpStatus.BAD_REQUEST, BangggoodCode.USER_INVALID_FORMAT, "유효하지 않은 이메일 형식입니다."),
 
     //Password
-    PASSWORD_INVALID_FORMAT(HttpStatus.BAD_REQUEST, "유효하지 않은 비밀번호 형식입니다."),
-    PASSWORD_HASHING_ERROR(HttpStatus.INTERNAL_SERVER_ERROR, "비밀번호 해싱 중 오류가 발생했습니다."),
+    PASSWORD_INVALID_FORMAT(HttpStatus.BAD_REQUEST, BangggoodCode.USER_INVALID_FORMAT, "유효하지 않은 비밀번호 형식입니다."),
+    PASSWORD_HASHING_ERROR(HttpStatus.INTERNAL_SERVER_ERROR, BangggoodCode.PASSWORD_HASHING_ERROR, "비밀번호 해싱 중 오류가 발생했습니다."),
 
     // Answer
-    ANSWER_INVALID(HttpStatus.BAD_REQUEST, "점수가 유효하지 않습니다."),
+    ANSWER_INVALID(HttpStatus.BAD_REQUEST, BangggoodCode.CHECKLIST_ERROR, "답변이 유효하지 않습니다."),
 
     // Checklist
-    CHECKLIST_COMPARISON_INVALID_COUNT(HttpStatus.BAD_REQUEST, "비교할 체크리스트 개수가 유효하지 않습니다."),
-    CHECKLIST_NOT_FOUND(HttpStatus.BAD_REQUEST, "체크리스트가 존재하지 않습니다."),
-    CHECKLIST_MEMO_INVALID_LENGTH(HttpStatus.BAD_REQUEST, "체크리스트 메모는 1000자 이하여야 합니다."),
-    CHECKLIST_NOT_OWNED_BY_USER(HttpStatus.BAD_REQUEST, "유저의 체크리스트가 아닙니다."),
+    CHECKLIST_NOT_FOUND(HttpStatus.BAD_REQUEST, BangggoodCode.CHECKLIST_NOT_FOUND, "체크리스트가 존재하지 않습니다."),
+    CHECKLIST_MEMO_INVALID_LENGTH(HttpStatus.BAD_REQUEST, BangggoodCode.CHECKLIST_ERROR, "체크리스트 메모는 1000자 이하여야 합니다."),
+    CHECKLIST_NOT_OWNED_BY_USER(HttpStatus.BAD_REQUEST, BangggoodCode.CHECKLIST_ERROR, "유저의 체크리스트가 아닙니다."),
```

Comment:
```suggestion
    CHECKLIST_NOT_OWNED_BY_USER(HttpStatus.BAD_REQUEST, BangggoodCode.UNAUTH_ERROR, "유저의 체크리스트가 아닙니다."),
```
![image](https://github.com/user-attachments/assets/67dca854-fee9-489d-a27e-23d3372e3bff)